docs/conf/agent/full.conf

```diff
@@ -0,0 +1,20 @@
+server {
+  metadata = "insert metadata"
+}
+
+plugins {
+  DataStore "sql" {
+    plugin_data {
+      drivername = "sqlite3"
+      filename = "./agentlocaldb" #TODO is this a good location
+    }
+  }
+
+  UserManagement "KeycloakAuth" {
+    plugin_data {
```

Comment:
Could we please put some comments here: 
```
# jwksURL - URL for JWKS verification
# here is a sample for Keycloak running locally on minikube:
jwksURL = "http://host.minikube.internal:8080/realms/tornjak/protocol/openid-connect/certs"
# for cloud deployment it would be something like: 
# jwksURL = "http://<ingress_access>/realms/tornjak/protocol/openid-connect/certs"
#
# redirectURL - URL for redirecting after successful authentication
# here is a sample for Keycloak running locally on minikube:
redirectURL = "http://localhost:8080/realms/tornjak/protocol/openid-connect/auth?client_id=Tornjak-React-auth"
# for a cloud deployment it would something like:
# redirectURL: "http://<ingress_access>/realms/tornjak/protocol/openid-connect/auth?client_id=Tornjak-React-auth"
```

Reply:
done!